src/plots/cartesian/layout_attributes.js

```diff
@@ -698,6 +698,15 @@ module.exports = {
             'In other cases the default is *hide past div*.'
         ].join(' ')
     },
+    drawminorticklabel: {
```

Comment:
I think this attribute could become `arrayOk` in future i.e. to label various minor ticks e.g. [-1, 0, 1], etc.
Let's add a comment about this in the code.
And for the attribute name I suggest we use something like `ticklabelindex` or `ticklabelids`.

Reply:
Okay, for the naming part I'd wait for you guys to make a final decision before adjusting the name everywhere.

Reply:
@LiamConnors
@alexcjohnson 
@emilykl 
What do you think about the attribute name suggested above?

Reply:
@archmoj just to clarify, your suggestion is to replace `drawminorticklabel` with either `ticklabelindex` or `ticklabelids` ?

Reply:
@LiamConnors Yes.

Reply:
`ticklabelindex` sounds good to me.

Reply:
@my-tien please rename the attribute to `ticklabelindex`.

Reply:
done